### DISCUSSION TITLE
Cutoff in plotEnrichHeatmap() and plotMarkerHeatmap()

### DISCUSSION THREAD

**Discussion by dimurali93** [GreenleafLab/ArchR#1903] [Questions / Documentation]
Hi All, 

Can I know what is Cutoff in plotEnrichHeatmap() and how to relate it to the cutOff = "FDR <= 0.01 & Log2FC >= 0.5" in plotMarkerHeatmap()

I got a list of enriched motifs from plotEnrichHeatmap() using a cutoff value = 5 and then i obtained positively correlated TFs. I would like to plot heatmap using plotMarkerHeatmap() function for the enriched motifs and was wondering what cutoff I should use so that I am neither losing any information in the process nor using arbitrary cutoffs at each step.

Thanks,
Divya

**Comment by rcorces** [comment]
They are unrelated. The parameters are both defined in their respective function documentation
https://www.archrproject.com/reference/plotEnrichHeatmap.html
